tgui/packages/tgui/interfaces/MarkdownViewer.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * @file MarkdownViewer.tsx
+ * @author ZephyrTFA
+ * @license MIT
+ */
```

Comment:
None of this should exist except maybe the MIT license header (I forget if the entire tgui folder is licensed under it or not). I know theo ther ones do but we don't do this anymore